hathor/sysctl/feature_activation/manager.py

```diff
@@ -0,0 +1,72 @@
+#  Copyright 2024 Hathor Labs
+#
+#  Licensed under the Apache License, Version 2.0 (the "License");
+#  you may not use this file except in compliance with the License.
+#  You may obtain a copy of the License at
+#
+#  http://www.apache.org/licenses/LICENSE-2.0
+#
+#  Unless required by applicable law or agreed to in writing, software
+#  distributed under the License is distributed on an "AS IS" BASIS,
+#  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#  See the License for the specific language governing permissions and
+#  limitations under the License.
+
+from hathor.feature_activation.bit_signaling_service import BitSignalingService
+from hathor.feature_activation.feature import Feature
+from hathor.sysctl.sysctl import Sysctl
+
+
+class FeatureActivationSysctl(Sysctl):
```

Comment:
Missing tests.

Reply:
Done in https://github.com/HathorNetwork/hathor-core/pull/962/commits/2fa908fd1cb31ad6f5b0c863d668509f5883b160

Reply:
Thanks!